src/components/CardGrid/CardGrid.test.js

```diff
@@ -38,7 +38,7 @@ describe('CardGrid testing', () => {
     filterSet.simulate('change', { target: { value: 'greenBlue' }})
     filterBtn.simulate('click')
 
-    expect(mockFn).toHaveBeenCalledTimes(2)
+    expect(mockFn).toHaveBeenCalledTimes(1)
```

Comment:
perhaps give filterBtn it's own unique mock function (I saw you passed mockFn to three different props).  Then you can be more certain that the filter mock function got called.  If you want to get really crazy, you could use .toHaveBeenCalledWith to test that it is getting the correct args as well.